db/migrate/20180215141222_change_org_links_default.rb

```diff
@@ -0,0 +1,8 @@
+class ChangeOrgLinksDefault < ActiveRecord::Migration
+  def up
+    change_column_default :orgs, :links, '{}' 
```

Comment:
the validation is preventing to change that value for the orgs (see https://github.com/DMPRoadmap/roadmap/blob/development/app/models/org.rb#L5). You should pass { org: []} instead.